src/lib/dataset/pagination.ts

```diff
@@ -24,3 +24,19 @@ export function numberOfPages(paginationContext: PaginationContext) {
 export function pageOffset(pagesize: number, pageno: number) {
   return Math.max(pageno - 1, 0) * pagesize;
 }
+
+/**
+ * Get page min/max row
+ */
+export function pageMinMax(paginationContext: PaginationContext) {
+  const pageRowMin = pageOffset(
+    paginationContext.pagesize,
+    paginationContext.pageno + 1,
+  ) + 1;
+  const pageRowMax = paginationContext.pageno === paginationContext.pagesize ? paginationContext.totalCount : pageRowMin + (paginationContext.pagesize - 1);
```

Comment:
This line feels a bit long-ish. I think a more traditional `if` / `else` would be more readable. Besides, I have serious doubts on the test being performed here: `pageno === pagesize`. What does `pageno` and `pagesize` have to do together?

Reply:
yeah you're right it was a mistake thx to spot it !
I will improve the readability as well